ethereum-network-partition-test/main_test.go

```diff
@@ -85,17 +84,23 @@ const (
 	retriesAttempts      = 20
 	retriesSleepDuration = 10 * time.Millisecond
 
-	newlineChar = "\n"
+	updateScriptStarlarkTemplate = `plan.update_service(service_id = "%s", config = UpdateServiceConfig(subnetwork = "%s"))`
+	headerStarlarkTemplate       = `def run(plan):`
+
+	noSerializedParams = ""
+	noDryRun           = false
 )
 
 var (
-	unblockedPartitionConnection = enclaves.NewUnblockedPartitionConnection()
-	blockedPartitionConnection   = enclaves.NewBlockedPartitionConnection()
-
 	nodeIds    = make([]int, numParticipants)
 	idsToQuery = make([]services.ServiceID, numParticipants)
 
 	isTestInExecution bool
+
+	healConnectionStarlark = fmt.Sprintf(`def run(plan):
```

Comment:
nit: maybe `allowConnectionStarlark` is more representative now that we're not talking in terms of partition, but in terms of connection